components/inputs/input-time.js

```diff
@@ -176,9 +152,9 @@ class InputTime extends LitElement {
 	getTime() {
 		const time = parseValue(this.value);
 		return {
-			hour: time.getHours(),
-			minute: time.getMinutes(),
-			second: time.getSeconds()
+			hours: time.getHours(),
```

Comment:
modified this for consistency. I have been using plural in other places to be consistent with javascript `Date` which has `getHours` etc.